pkg/kp/kv.go

```diff
@@ -50,6 +53,42 @@ func (err KVError) Error() string {
 	return fmt.Sprintf("%s failed for path %s", err.Op, err.Key)
 }
 
+func (c consulStore) PutHealth(res map[string]string) (time.Time, time.Duration, error) {
+	key := healthPath(res["service"], res["node"])
```

Comment:
Magic keys are a really bad idea - it makes it impossible to statically check callers correctness of this program. This should at the very least be a struct with named fields that are marshaled in a specific way.